docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>It's not clear what is the purpose of <code>serialization.to_state_dict()</code>  here - could you elaborate?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>I wanted to demonstrate the two provided formats for serialization (dict and bytes). But I didn't have further thoughts on those, I'd take any suggestion to make it more interesting though!</p>

Reply:
I meant : in this example `serialization.to_state_dict(optimizer.target)` is exactly the same as `optimizer.target.unfreeze()`. But `serialization.to_state_dict()` can do more than that (e.g. if you say `serialization.to_state_dict(optimizer)`.

Reply:
<p>Ah, oh! Got it now indeed, will change the example for something different!</p>